src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,87 +4,96 @@ short-title: Integrate Flutter
 description: Learn how to integrate a Flutter module into your existing iOS project.
 ---
 
-TODO(jmagman): add the pre-built framework alternative and review.
+Flutter can be incrementaly added into your existing iOS application as embedded
+frameworks.
 
```

Comment:
Do we need to mention bitcode or the min OS version?

Reply:
Fortunately the bitcode bugs seem fixed so they don't have to turn it off anymore.  
I will add a systems requirements section.